tests/test_authorization_header.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from tests.test_case import TestCase
+from app import decode_authorization_header
+from eth_keys import keys
+import base64
+
+
+class TestAuthorizationHeader(TestCase):
+    def sign_message(self, message):
+        pk = keys.PrivateKey(b'\x01' * 32)
+        signature = pk.sign_msg(message)
+        signature_bytes = signature.to_bytes()
+        public_address = pk.public_key.to_checksum_address()
+        return signature_bytes, public_address
```

Comment:
`sign_message` is duplicated in `TestApi` - this method looks pretty complex.
Maybe it makes sense to extract it somewhere, i.e. in `signature.py` or in `tests/helpers.py`?